package.json

```diff
@@ -58,9 +58,6 @@
     "optimize-css-assets-webpack-plugin": "^3.2.0",
     "ora": "^1.2.0",
     "portfinder": "^1.0.13",
-    "postcss-import": "^11.0.0",
```

Comment:
Why is this not needed?

Reply:
`postcss-import` – Imports that are used in our `<style lang="less">` are supported by Less pre-compiler

`postcss-url` – I don't think we use any feature this plugin provides

`postcss-loader` – this one might actually be used and I'll put it back if that is true

I was removing this because [of this](https://github.com/celtra/mab/pull/9912/commits/3a7a9eb625df634d29e4e60163628484745bf175) commit where I had to install both `postcss-import` and `postcss-url` packages because of `.postcssrc.js` file. Then I tried removing post css all together and everything seemed to work, but I might be completely wrong. Do you know where we might need it?